apps/aevm/test/aevm_test_utils.erl

```diff
@@ -119,6 +120,23 @@ validate_storage(State, #{exec := #{address := Addr}} = Spec) ->
         _ -> true
     end.
 
+validate_log(State, #{logs := Logs} =_Spec) ->
+    GeneratedLogs = aevm_eeevm_state:logs(State),
+    RLPLogs = evm_encode_log(GeneratedLogs, []),
+    ?assertEqual(Logs, logs_to_string( aec_hash:hash(evm,RLPLogs)));
+validate_log(_,_) -> true.
+
+evm_encode_log([], Acc) ->  aeu_rlp:encode(lists:reverse(Acc));
+evm_encode_log([{<<Address:256>>, Topics, Data}|Rest], Acc) ->
+    %% The EVM tests expects 160 bit addresses and
+    %% RLP expects tuples as lists.
+    evm_encode_log(Rest, [[<<Address:160>>, Topics, Data] | Acc]).
```

Comment:
I see [sample test](https://github.com/ethereum/tests/blob/3f5febc901913ef698f1b09dda8705babd729e4a/src/VMTestsFiller/vmLogTest/log0_emptyMemFiller.json#L12) has address of 40 hex (i.e. 20 bytes i.e. 160 bits), code keeps that field as [int](https://github.com/aeternity/epoch/blob/1bdc3a9cf7a37fb3b2199412a2da80eb90acd1fb/apps/aevm/src/aevm_eeevm.erl#L60), code creates log entry using `aevm_eeevm_state:address` (see `aevm_eeevm.erl`), so at this point in the test restoring from 256 to 160 bits is lossless, and the test data already knows that input value. Ok.